### ISSUE TITLE
Feature Request: Auto update site credential in Vault after user changes it

### ISSUE THREAD

**Issue by Moxville** [bitwarden/browser#291]
Hi Kyle,
When user changes his credentials (username / password) for a site, bitwarden shall auto update the credentials in the vault and display a message

This is a very useful feature in my opinion, so requesting to add this in a future release

**Comment by Primokorn** [comment]
+1 for the auto update but there's no need to add a new notification IMHO.

**Comment by GeoDosch** [comment]
If this were implemented, it should be done so as an option.  There are sites where I have more than one user name, each with its own password, so having the username in bitwarden change would be a really bad thing.  Also, IMHO, it's unusual that a username would be changed for a site (as compared to a password), so in that rare case I don't have a problem manually editing the bitwarden entry.

**Comment by jpbaril** [comment]
Yes, it should be semi-auto. The extension should ask if we want to update the site credentials. And if we have more than one site match, then ask which site to update.

**Comment by stderr-to-devnull** [comment]
"thank you" for requiring us to create yet another user on another external forum since github apparently isn't good enough...
btw this is still not working as of today
